pydicom/config.py

```diff
@@ -421,9 +421,10 @@ def future_behavior() -> None:
     :attr:`~pydicom.config.INVALID_KEY_BEHAVIOR
 
     """
-    global _use_future
+    global _use_future, INVALID_KEYWORD_BEHAVIOR
```

Comment:
You might be able to use `nonlocal` here instead for `INALID_KEYWORD_BEHAVIOR`

Reply:
Hmm ... I hadn't even heard of `nonlocal` before - but looking it up, it seems to be useful for the case where you might use the same name at a global level and also in some nested scopes, giving the ability to differentiate between the two.  I think `global` is the more natural here because it is defined at that level.